src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -927,9 +927,14 @@ and, optionally,
 function air_temperature_from_liquid_ice_pottemp(θ_liq_ice::FT, ρ::FT,
   q::PhasePartition{FT}=q_pt_0(FT)) where {FT<:Real}
 
-  cv = cv_m(q)
+  cvm = cv_m(q)
+  cpm = cp_m(q)
   R_m = gas_constant_air(q)
-  return θ_liq_ice * (ρ*R_m*θ_liq_ice/FT(MSLP))^(R_m/cv) + latent_heat_liq_ice(q)/cv
+  κ = 1 - cvm/cpm
+  T_O0 = (ρ*R_m*θ_liq_ice/FT(MSLP))^(R_m/cvm)*θ_liq_ice
```

Comment:
I find the O and 0 in the subscript confusing. Why not something like:
```suggestion
  T_u = (ρ*R_m*θ_liq_ice/FT(MSLP))^(R_m/cvm)*θ_liq_ice
```
(for unsaturated) and replace `T_01` by `T_1` etc.?

Reply:
Sure, we can do that. I had written it as `O` indicating `Order_1`, `Order_2`, etc. But `T_1`, `T_2` works fine too.